Automattic/wp-calypso
# Split Product store styling into separate component styling.

### Description
To prevent the current `product-store/style.scss` getting bloated. It is better to split it into multiple css files per component. This way it is a lot easier for us to find relevant styling when we modify the component.

#### Proposed Changes

* [Move 'Jetpack Free' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/5c7c45f4dc1bc2d225ade219b62a473fd51fd5b4)
* [Move 'Need more info' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/93c40ebf481b5ba8d177952cebd5cb60186dce00)
* [Move 'Recommendations' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/4b5aa8551ecf843fa1af1c4e5eb5978aa4bfe2b0)
* [Create 'Pricing Banner' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/65f977c52409e1c8869bc5ba6417cb49ec52d607)
* [Move 'See all features' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/6c7e2e959d1e12694a60846c48b37357b7349834)
* [Move 'View filter' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/40ffa44cccebb85593e4564d26dabce302d2da1b)
* [Move 'Item list' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/1ee40426fb70c425b8b4f1b5ba95d4b53a33e003)
* [Move 'Most popular' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/c2ffe7c7019dab44d777b4791646cd04d66520bd)
* [Move 'Feature list' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/356e546757ad1987adbdaffabeee3b0804268109)
* [Move 'Bundle list' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/6a91175252a5df8e11b76bad5f50f2c515553c57)
* [Move 'Product list' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/20f1d67ed979186d4b77458973a3d6ed79100e48)
* [Move 'Item price' component and styling to a separate folder.](https://github.com/Automattic/wp-calypso/commit/66f0b11e785884b6d6fd2ad54b0b81621bf42277)

#### Testing Instructions



* Run `git fetch && git checkout refactor/split-product-store-style-scss`
  * Run `yarn start-jetpack-cloud`
* Go to http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1 or use the jetpack cloud live link and goto `/pricing?flags=jetpack/pricing-page-rework-v1`.
* Confirm no styling has been broken with the code refactoring.

#### Pre-merge Checklist






Related to 1202796695664022-as-1202911895278960

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202911895278960
